templates/common/azure/files/chrony.yaml

```diff
@@ -0,0 +1,50 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/chrony.conf"
```

Comment:
Hmm, it's unfortunate that chrony doesn't seem to support something like `/etc/chrony.conf.d`. Is the only real change here from the default this line:

> `refclock PHC /dev/ptp0 poll 3 dpoll -2 offset 0`

?

Might be worth highlighting that better in the config. Maybe e.g. putting it up near the top right after the MCO-specific comment?

Reply:
yes,that's the main change. I kept at bottom as this was the addition. I will add it on top if that makes more sense.

Reply:
fixed